include/model/CResourceMonitor.h

```diff
@@ -49,6 +49,12 @@ class MODEL_EXPORT CResourceMonitor {
         core_t::TTime s_BucketStartTime;
         std::size_t s_BytesExceeded;
         std::size_t s_BytesMemoryLimit;
+        std::size_t s_CategorizedMessages;
+        std::size_t s_TotalCategories;
+        std::size_t s_FrequentCategories;
+        std::size_t s_RareCategories;
+        std::size_t s_DeadCategories;
+        model_t::ECategorizationStatus s_CategorizationStatus;
```

Comment:
I know you're only extending, but should this provide defaults? It seems like this object is a recipe for uninitialised variables as it stands.